### DISCUSSION TITLE
How to access to the big window configuration view?

### DISCUSSION THREAD

**Discussion by Pedro-vk** [melianmiko/OpenFreebuds#36] [Q&A] [answered]
Hi, I saw that you have a big configuration screen.
I'm using Ubuntu and I only see a small screen with some configuration.
How can I open the big one?

Thanks!
![image](https://github.com/user-attachments/assets/59d244c9-2c40-46fd-8f32-1eea78c71e04)

**Comment by melianmiko** [comment, marked as answer]
That's because you're using OpenFreebuds 0.13.

In next version (0.14), application was rewritten to Qt and redesigned from scratch. So, this new settings window will be available after v0.14 release, approximately at next week. If you want to try it now, you could grab test builds from [github actions artifacts](https://github.com/melianmiko/OpenFreebuds/actions/runs/11072534615), it will also include your translations.

**Reply by Pedro-vk** [reply]
I was having an error running it locally, but when I tested it again to ask you for it using `poetry run python -m openfreebuds_qt -vcs` it worked :joy: 
I'll push a new PR with a couple of fixes, now I'm able to check it properly.
